lib/matplotlib/backend_bases.py

```diff
@@ -2040,12 +2044,45 @@ def draw_idle(self, *args, **kwargs):
             with self._idle_draw_cntx():
                 self.draw(*args, **kwargs)
 
+    @property
+    def pixel_ratio(self):
```

Comment:
perhaps just spell it out fully as `device_pixel_ratio`, consistently with GUI toolkits?  (or possibly `dpi_ratio` (just with no leading underscore))